api/loaders/graph_loader.py

```
@@ -107,7 +107,7 @@
 
                     embedding_result = embedding_model.embed(batch)
                     embed_columns.extend(embedding_result)
-            except Exception as e:
+            except Exception as e:  # pylint: disable=broad-exception-caught
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)